harmony/util.py

```diff
@@ -162,6 +165,10 @@ def bool_envvar(name: str, default: bool) -> bool:
         value = environ.get(name)
         return str.lower(value) == 'true' if value is not None else default
 
+    def int_envvar(name: str, default: bool) -> int:
```

Comment:
Why is `default` a `bool` and not an `int`?

Reply:
Yikes, that was a mistake. Thanks for catching it.